content/en/docs/tasks/job/job-with-pod-to-pod-communication.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Job with Pod-to-Pod Communication
+content_type: task
+min-kubernetes-server-version: v1.21
+weight: 30
+---
+
+<!-- overview -->
+
+In this example, we will run a Job in [Indexed completion mode](https://kubernetes.io/blog/2021/04/19/introducing-indexed-jobs/) configured such that
+the pods created by the Job can communicate with each other using pod hostnames rather than pod IPs.
+
+Pods within a Job might need to communicate among themselves. They could query the Kubernetes API
```

Comment:
Changed to `The user workload running in each pod could query the Kubernetes API server...` since it is not the pod itself which can query the k8s api server, but rather the user's application code executing inside the pod, which they would modify to use to use a k8s client library to query the API server.